docs/rules/no-cb-test.md

```diff
@@ -0,0 +1,26 @@
+# Ensure no `test.cb()` is used
+
+Disallow the use of `test.cb()`.
```

Comment:
Couldn't think of a good description, but might be nice to have more explanations.